spec/lib/tasks/annual_fund_impact_metrics_activities_spec.rb

```diff
@@ -1,21 +1,21 @@
 require "tempfile"
 
 RSpec.describe "rake activities:annual_fund_impact_metrics", type: :task do
```

Comment:
Don't want to make work on a non-project day, but is it worth adding a fund exclusion test, like you do for the `programme_status` exclusions?

Reply:
This is already captured in the existing specs, the factories create a load of funds, so I thought that would be sufficient